mod-source-record-manager-server/src/test/java/org/folio/verticle/consumers/DataImportJournalConsumerVerticleTest.java

```diff
@@ -101,167 +95,6 @@ public void setUp() {
-      .withTenant(TENANT_ID)
-      .withOkapiUrl(OKAPI_URL)
-      .withToken(TOKEN)
-      .withContext(new HashMap<>() {{
-        put(ITEM.value(), recordJson.encode());
-        put(MARC_BIBLIOGRAPHIC.value(), recordJson.encode());
-      }})
-      .withProfileSnapshot(new ProfileSnapshotWrapper()
-        .withId(UUID.randomUUID().toString())
-        .withContentType(ACTION_PROFILE)
-        .withContent(JsonObject.mapFrom(new ActionProfile().withFolioRecord(ActionProfile.FolioRecord.ITEM))))
-      .withCurrentNode(new ProfileSnapshotWrapper()
-        .withId(UUID.randomUUID().toString())
-        .withContentType(ACTION_PROFILE)
-        .withContent(JsonObject.mapFrom(new ActionProfile().withFolioRecord(ActionProfile.FolioRecord.ITEM))))
-      .withEventsChain(List.of(DI_INVENTORY_HOLDING_CREATED.value(), DI_INVENTORY_ITEM_CREATED.value()));
-
-    // when
-    KafkaConsumerRecord<String, String> kafkaConsumerRecord = buildKafkaConsumerRecord(completedEventPayload);
-    dataImportJournalKafkaHandler.handle(kafkaConsumerRecord);
-
-    // then
-    Future<JobExecutionLogDto> future = journalRecordDao.getJobExecutionLogDto(jobExecution.getId(), TENANT_ID);
-    future.onComplete(ar -> {
-      if (ar.succeeded()) {
-        context.assertTrue(ar.succeeded());
-        Assert.assertNotNull(ar.result());
-      }
-    });
-    async.complete();
-  }
-
-  @Test
-  public void testJournalErrorAction(TestContext context) throws IOException, ExecutionException, InterruptedException {
-    Async async = context.async();
-
-    // given
-    DataImportEventPayload eventPayload = new DataImportEventPayload()
-      .withEventType(DI_ERROR.value())
-      .withTenant(TENANT_ID)
-      .withOkapiUrl(OKAPI_URL)
-      .withToken(TOKEN)
-      .withContext(new HashMap<>() {{
-        put(HOLDINGS.value(), recordJson.encode());
-        put(MARC_BIBLIOGRAPHIC.value(), recordJson.encode());
-        put("ERROR", "java.lang.IllegalArgumentException: Can not handle event payload");
-      }})
-      .withProfileSnapshot(new ProfileSnapshotWrapper()
-        .withId(UUID.randomUUID().toString())
-        .withContentType(ACTION_PROFILE)
-        .withContent(JsonObject.mapFrom(new ActionProfile().withFolioRecord(ActionProfile.FolioRecord.HOLDINGS))))
-      .withCurrentNode(new ProfileSnapshotWrapper()
-        .withId(UUID.randomUUID().toString())
-        .withContentType(ACTION_PROFILE)
-        .withContent(JsonObject.mapFrom(new ActionProfile().withFolioRecord(ActionProfile.FolioRecord.HOLDINGS))))
-      .withEventsChain(List.of(DI_SRS_MARC_BIB_RECORD_CREATED.value(), DI_INVENTORY_HOLDING_CREATED.value()));
-
-    // when
-    KafkaConsumerRecord<String, String> kafkaConsumerRecord = buildKafkaConsumerRecord(eventPayload);
-    dataImportJournalKafkaHandler.handle(kafkaConsumerRecord);
-
-    // then
-    Future<JobExecutionLogDto> future = journalRecordDao.getJobExecutionLogDto(jobExecution.getId(), TENANT_ID);
```

Comment:
I think we don't need to delete these test methods, just need to replace getJobExecutionLogDto() to getByJobExecutionId()

Reply:
Hi @SerhiiNosko - Updated, Thank you 🙂